### PR TITLE
Add maximum values to permissions reference doc

### PR DESCRIPTION
**Pull request by Jackson-Woods**
Added maximum values each app type can request and consent

### CODE REVIEW

File: concepts/permissions-reference.md [hunk 1]
```diff
@@ -27,12 +27,23 @@ The _constraint_ element of the name determines the potential extent of access y
...
+| AzureADMyOrg                       | Users from the organization where the app is registered  | 400                                       | 400                                                       | About 155 delegated permissions and about 300 application permissions |
+| AzureADMultipleOrgs                | Users from any Azure AD organization                     | 400                                       | 400                                                       | About 155 delegated permissions and about 300 application permissions |
+| PersonalMicrosoftAccount           | Consumer users (Outlook.com or Live.com accounts, etc.)  | 30                                        | 30                                                        | 30                                                                    |
+| AzureADandPersonalMicrosoftAccount | Consumer users and users from any Azure AD organization  | 30                                        | 30                                                        | 30                                                                    |
```
Comment (FaithOmbongi) [inline concepts/permissions-reference.md:45]: Suggestion: is there a chance of the max. limits being less than 155/300 or can we avoid the uncertainty of using "about"?
```suggestion
| AzureADMyOrg                       | Users from the organization where the app is registered  | 400                                       | 400                                                       | 155 delegated permissions and 300 application permissions |
| AzureADMultipleOrgs                | Users from any Azure AD organization                     | 400                                       | 400                                                       | 155 delegated permissions and 300 application permissions |
| PersonalMicrosoftAccount           | Consumer users (such as Outlook.com or Live.com accounts)  | 30                                        | 30                                                        | 30                                                                    |
| AzureADandPersonalMicrosoftAccount | Consumer users and users from any Azure AD organization  | 30                                        | 30                                                        | 30                                                                    |
```

**Review by FaithOmbongi — APPROVED** [review]
Thanks @Jackson-Woods for working on this change.

Left a comment on the limits, otherwise approved.

I think we should also link from the [application](https://docs.microsoft.com/en-us/graph/api/resources/application?view=graph-rest-1.0) resource doc (descriptions of requiredResourceAccess and signInAudience) to the section you've added.

File: concepts/permissions-reference.md (hunk 1 above)
Comment (Jackson-Woods) [reply concepts/permissions-reference.md:45]: 155 and 300 are relatively conservative values. Usually the limit will be slightly higher. It depends on the specific permissions passed, unfortunately.

**Comment by FaithOmbongi** [comment]
Heads up @Jackson-Woods, I believe we'd agreed that the PR was ready to merge. I've added the **ready to merge** label to move it into the merge queue. If there's any reason we should hold back this PR, please replace **ready to merge** with **do not merge** label.

**Pull request merged**